packages/front-end/components/Features/AttributeModal.tsx

```diff
@@ -65,52 +68,51 @@ export default function AttributeModal({ close, attribute }: Props) {
           value.hashAttribute = false;
         }
 
-        const attributeSchema = [...schema];
-
-        // Editing
-        if (attribute) {
-          const i = schema.findIndex((s) => s.property === attribute);
-          if (i >= 0) {
-            attributeSchema[i] = value;
-          } else {
-            attributeSchema.push(value);
-          }
-        }
-        // Creating
-        else {
-          attributeSchema.push(value);
-        }
-
         // Make sure this attribute name doesn't conflict with any existing attributes
-        if (
-          attributeSchema.filter((s) => s.property === value.property).length >
-          1
-        ) {
+        if (!attribute && schema.some((s) => s.property === value.property)) {
```

Comment:
The form below allows changing this for existing attributes, so we also need to check that the name they are changing to does not conflict with another property.